app/src/main/java/ch/nevis/exampleapp/coroutines/domain/model/response/TransactionConfirmationResponse.kt

```diff
@@ -0,0 +1,33 @@
+/**
+ * Nevis Mobile Authentication SDK Example App
+ *
+ * Copyright © 2024. Nevis Security AG. All rights reserved.
+ */
+
+package ch.nevis.exampleapp.coroutines.domain.model.response
+
+import ch.nevis.mobile.sdk.api.localdata.Account
+
+/**
+ * [Response] class that indicates an account selection has to be started.
```

Comment:
I think this description is wrong. It is about account selection instead of transaction confirmation response.

Reply:
Fixed in 9396c816e5fdd7a55b575e81cc8e29de14c5260b